From 70d9041ea21d8f27afe6298ca7a3ab54fdee98eb Mon Sep 17 00:00:00 2001
From: Pardha Kesapragada <pkesapragada@habana.ai>
Date: Fri, 05 Mar 2021 13:52:58 +0200
Subject: [PATCH] [SW-20154] gaudi2_arc_gnu: remove todo

There is no HW FSM bit setting needed for Gaudi2.
Hence remove the TODO

Change-Id: If9123ef36dce7b73c3038688a161015d9b10b773
Signed-off-by: Pardha Kesapragada <pkesapragada@habana.ai>
---

diff --git a/gaudi2_arc_gnu/common.c b/gaudi2_arc_gnu/common.c
index 7e8cb41..eb0c362 100644
--- a/gaudi2_arc_gnu/common.c
+++ b/gaudi2_arc_gnu/common.c
@@ -82,7 +82,7 @@
 
 void enable_flr_handling_fsm(void)
 {
-	/* TODO: No HW FSM bits in Gaudi2, wait for ORON to comeback */
+	/* No HW FSM bits in Gaudi2, No need to set anything */
 }
 
 static int verify_sha(uint32_t data_size, unsigned char *ptr_to_data,
